irs-edc-client/src/test/java/org/eclipse/tractusx/irs/edc/client/contract/service/EdcContractAgreementServiceTest.java

```diff
@@ -136,7 +134,7 @@ void shouldReturnContractAgreementNegotiation() {
 
         //THEN
         Mockito.verify(restTemplate)
-               .getForEntity("/v2/contractagreements/contractAgreementId/negotiation",
+               .getForEntity("https://irs-consumer-controlplane.dev.demo.catena-x.net/data/management/v2/contractagreements/contractAgreementId/negotiation",
```

Comment:
Please use another domain not catena-x.net this might be misleading in the future as this is a mock you can use anything.